emgapianns/management/lib/sanity_check.py

```diff
@@ -6,19 +6,40 @@
 
 from emgapianns.management.lib.uploader_exceptions import NoAnnotationsFoundException, \
     UnexpectedLibraryStrategyException, QCNotPassedException, CoverageCheckException
-from emgapianns.management.lib.utils import read_chunkfile
+from emgapianns.management.lib import utils
 from emgapianns.management.webuploader_configs import get_downloadset_config
+from backlog import models as backlog_models
 
 logger = logging.getLogger(__name__)
 
 
+def get_result_status(accession):
+    """
+    no_qc: upload qc results only. No downloads
+    no_cds: upload qc and taxonomy only
+    no_tax: standard upload (tax files are optional)
+    no_tax and no_cds: upload qc. processed reads and other ncRNA downloadable
+
+    :param accession:
+    :return: result status
+    """
+    result_status = backlog_models.AnnotationJob.objects.using('backlog_dev').all().filter(
```

Comment:
No need to .all().filter(), you can just do .filter()

Also, I think you you check if there are results (results_status[0] could break if there are not results)

The backlog_dev value is guess is for testing